src/buildMathML.js

```diff
@@ -149,11 +150,25 @@ groupTypes.ordgroup = function(group, options) {
 };
 
 groupTypes.text = function(group, options) {
-    const inner = buildExpression(group.value.body, options);
-
-    const node = new mathMLTree.MathNode("mtext", inner);
-
-    return node;
+    const body = group.value.body;
+    const inner = [];
+    let currentText = null;
+    for (let i = 0; i < body.length; i++) {
+        const group = buildGroup(body[i], options);
+        if (group.type === 'mtext' && currentText != null) {
+            Array.prototype.push.apply(currentText.children, group.children);
+        } else {
+            inner.push(group);
+            if (group.type === 'mtext') {
+                currentText = group;
+            }
+        }
+    }
```

Comment:
This loop combines multiple adjacent `<mtext>` tags into a single `<mtext>` tag.  I'll add a comment to this effect.